pkg/kubecost/asset.go

```diff
@@ -2930,6 +2930,7 @@ func (as *AssetSet) accumulate(that *AssetSet) (*AssetSet, error) {
 }
 
 type AssetSetRange struct {
+	FromStore string
```

Comment:
It doesn't look like this made it into the codec, is that of concern?

Reply:
Good point, I should probably add it, just to be safe! It likely wouldn't matter too much (just a tool for determining what build/run uses as a source) but it could be confusing to have it absent.

Reply:
We were having some trouble with it yesterday, and my haphazard upgrading of it. I am not sure if @AjayTripathy will want two releases in a row with upgrades, but I am not sure if not including it will cause problems.

Reply:
Well, it's getting updated! We'll roll with the punches. My codec generated and is working fine on v0.0.4.